doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,37 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with two classes which can be used to process electron holography data:
+
+* :py:class:`~._signals.hologram_image.HologramImage`

```

Comment:
`HologramImage` shouldn't be listed here. In all chances this PR will be merged for 1.0, but the fate of #1079 is not so clear.